docs/enterprise/index.md

```diff
@@ -0,0 +1,99 @@
+- [Introduction to Quorum: Blockchain for the Financial Sector](https://medium.com/blockchain-at-berkeley/introduction-to-quorum-blockchain-for-the-financial-sector-58813f84e88c)
+
+
+## Organizations
+
+Some collaborative efforts to make Ethereum enterprise friendly have been put together by different organizations:
+
+- [EEA](https://entethalliance.org/)  
+*The Enterprise Ethereum Alliance is a member-driven standards organization whose charter is to develop open, blockchain specifications that drive harmonization and interoperability for businesses and consumers worldwide. Our global community of members is made up of leaders, adopters, innovators, developers, and businesses who collaborate to create an open, decentralized web for the benefit of everyone.*
+
+- [Hyperledger Foundation](https://hyperledger.org)  
+*Hyperledger is an open source collaborative effort created to advance cross-industry blockchain technologies. It is a global collaboration, hosted by The Linux Foundation, including leaders in finance, banking, Internet of Things, supply chains, manufacturing and Technology.*
+*The foundation has some projects in it that work with the Ethereum stack:*
+    - [Hyperledger Besu](https://www.hyperledger.org/blog/2019/08/29/announcing-hyperledger-besu)
+    - [Hyperledger Burrow](https://www.hyperledger.org/projects/hyperledger-burrow)
+    - [Hyperledger Sawtooth](https://www.hyperledger.org/projects/sawtooth)
+
+## Enterprise Focused Services
+
+The following projects provide blockchain services for enterprises grade systems:
+
+- [Blockapps](https://blockapps.net/) *implementation of the Enterprise Ethereum protocol, tooling and APIs that form the STRATO platform*
+- [Clearmatics](https://www.clearmatics.com/about) *protocols and peer-to-peer platform architecture, blockchain R&D company*
+- [PegaSys Plus](https://pegasys.tech/enterprise/) *offers the same features and functionalities as HF Besu, plus additional enterprise focused benefits*
+- [Quorum](https://www.goquorum.com/) *open source blockchain platform that combines the innovation of the public Ethereum community with enhancements to support enterprise needs*
+
+## Protocol and Infrastructure
+
+- [Hyperledger Besu](https://www.hyperledger.org/projects/besu) *open-source Ethereum client developed under the Apache 2.0 license and written in Java*
+- [Hyperledger Burrow](https://kaleido.io/) *modular blockchain client with a permissioned smart contract interpreter partially developed to the specification of the Ethereum Virtual Machine (EVM)*
+- [Infura](https://infura.io/) *scalable API access to the Ethereum and IPFS networks*
+- [Kaleido](https://kaleido.io/) *full-stack platform for building and running cross-cloud, hybrid enterprise ecosystems*
+- [Hyperledger Sawtooth](https://www.hyperledger.org/projects/sawtooth) *modular platform for building, deploying, and running distributed ledgers.*
```

Comment:
I recommend removing this for now until it's clear this is an Ethereum project. We want to avoid creating confusion.

Reply:
I'm sure its an Enterprise Ethereum project, but I've gone ahead and removed the mentions for now.